lib/applications/resolvers.ts

```diff
@@ -541,32 +546,24 @@ export const createExternalRenewalApplication: Resolver<
         processingFee: process.env.PROCESSING_FEE,
         donationAmount: 0, // ? Investigate
         paymentMethod: 'SHOPIFY',
-        // TODO: Replace shipping info with Shopify checkout inputs
+        // Set shipping address to be same as home address by default
         shippingAddressSameAsHomeAddress: true,
-        shippingFullName: formatFullName(
-          applicant.firstName,
-          applicant.middleName,
-          applicant.lastName
-        ),
-        shippingAddressLine1: applicant.addressLine1,
-        shippingAddressLine2: applicant.addressLine2,
-        shippingCity: applicant.city,
-        shippingProvince: applicant.province,
-        shippingCountry: applicant.country,
-        shippingPostalCode: stripPostalCode(applicant.postalCode),
-        // TODO: Replace billing info with Shopify checkout inputs
+        shippingFullName: null,
+        shippingAddressLine1: null,
+        shippingAddressLine2: null,
+        shippingCity: null,
+        shippingProvince: null,
+        shippingCountry: null,
+        shippingPostalCode: null,
```

Comment:
How come shipping address is changed to being all null here? I also see that it's not being updated from shopify webhook. When would shipping address data be populated?

Reply:
Shipping address fields should all be `null` when shipping address is the same as home address. It won't get updated because the Shopify webhook payload does not contain any shipping info. We default the shipping address to be the same as home address.

Reply:
It is okay to set the shipping address fields to `null` because the `flattenApplication` function in `lib/applications/utils.ts` populates billing/shipping address fields when they are the same as home address